webhooks/wh-general.md

```diff
@@ -1,12 +1,16 @@
 # General Webhooks
 
-This is a newer form of Webhook message which will be extended in future to carry events about different entities in Mews.
+This is a newer form of Webhook message which carries event information related to different entities in Mews.
 
 Each General Webhook message corresponds to a single property or enterprise integration, identified by `EnterpriseId` and `IntegrationId` in the request body.
 You will receive events for all the properties you are integrated with, and can identify each one from this information.
 
 The message encapsulates all events occurring at the same time which belong to event types to which you have subscribed.
-For example, if you are interested in reservations you will subscribe to `ServiceOrderUpdated` events (a reservation is a type of Service Order) and you may receive multiple `ServiceOrderUpdated` events in one Webhook message. When the entity is added you may receive event for both added and update in one webhook message.
+For example, if you are interested in reservations you will subscribe to `ServiceOrderUpdated` events (a reservation is a type of Service Order) and you may receive multiple `ServiceOrderUpdated` events in one Webhook message.
```

Comment:
I'd still mention in a some way this part: 
>When the entity is added you may receive event for both added and update in one webhook message.

As we first had only `Updated` event which also were triggered for newly added entities we still keep this behavior and as we now added a `Added` event there could be events of both types with the same Id.

Reply:
Following our meeting to discuss this, thank you @jirrick I now understand the significance of this point! As a result, I'm going to suggest some new wording to address this - a new section called 'Added vs Updated events'